tests/tensor/rewriting/test_linalg.py

```diff
@@ -545,3 +546,15 @@ def test_svd_uv_merge():
             assert node.op.compute_uv
             svd_counter += 1
     assert svd_counter == 1
+
+
+def test_inv_inv_rewrite():
+    x = pt.matrix("a")
+    ii_x = pt.linalg.inv(pt.linalg.inv(x))
+    f_rewritten = function([x], ii_x, mode="FAST_RUN")
+    nodes = f_rewritten.maker.fgraph.apply_nodes
+
+    assert all(isinstance(node.op, DeepCopyOp) for node in nodes)
```

Comment:
I suggest you just check that there's no `inverse_op` in the graph, rather than looking for the copy explicitly